From e1ca71562204bc622e6620414a4954cbedcd688e Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Tue, 18 May 2010 19:31:38 -0300
Subject: [PATCH 02/20] doc: Fix host forwarding monitor command documentation

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1274211101-5748-2-git-send-email-armbru@redhat.com>
Patchwork-id: 9376
O-Subject: [RHEL-6 KVM PATCH 1/4] doc: Fix host forwarding monitor command
	documentation
Bugzilla: 588719
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

Commit f3546deb replaced host_net_redir by hostfwd_add,
hostfwd_remove, but neglected to update documentation.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
(cherry picked from commit 21413d683b1fa264d82b6a3408ec8f2b5ce29d27)
---
 qemu-monitor.hx |   13 ++++++++++---
 1 files changed, 10 insertions(+), 3 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-monitor.hx |   13 ++++++++++---
 1 files changed, 10 insertions(+), 3 deletions(-)

diff --git a/qemu-monitor.hx b/qemu-monitor.hx
index c7ae741..b8727c1 100644
--- a/qemu-monitor.hx
+++ b/qemu-monitor.hx
@@ -952,7 +952,14 @@ ETEXI
         .help       = "redirect TCP or UDP connections from host to guest (requires -net user)",
         .mhandler.cmd = net_slirp_hostfwd_add,
     },
+#endif
+STEXI
+@item hostfwd_add
+@findex hostfwd_add
+Redirect TCP or UDP connections from host to guest (requires -net user).
+ETEXI
 
+#ifdef CONFIG_SLIRP
     {
         .name       = "hostfwd_remove",
         .args_type  = "arg1:s,arg2:s?,arg3:s?",
@@ -963,9 +970,9 @@ ETEXI
 
 #endif
 STEXI
-@item host_net_redir
-@findex host_net_redir
-Redirect TCP or UDP connections from host to guest (requires -net user).
+@item hostfwd_remove
+@findex hostfwd_remove
+Remove host-to-guest TCP or UDP redirection.
 ETEXI
 
     {
-- 
1.7.0.3

